engshell.py

```diff
@@ -61,112 +77,128 @@ def LLM(prompt, mode='text'):
         raise ValueError(f'prompt ({prompt}) flagged by moderation endpoint')
     time.sleep(1.0/API_CALLS_PER_MIN)
     if mode == 'text':
-        messages=[
+        messages = [
             {"role": "system", "content": CONGNITIVE_SYSTEM_CALIBRATION_MESSAGE},
             {"role": "user", "content": prompt},
         ]
     elif mode == 'code':
-        messages=memory+[{"role": "user", "content": prompt}]
+        messages = memory+[{"role": "user", "content": prompt}]
     elif mode == 'install':
-        messages=[
+        messages = [
             {"role": "system", "content": INSTALL_SYSTEM_CALIBRATION_MESSAGE},
             {"role": "user", "content": prompt},
         ]
     response = openai.ChatCompletion.create(
-      #model="gpt-4",
-      model="gpt-3.5-turbo-0301",
-      messages=messages,
-      temperature = 0.0
+        # model="gpt-4",
+        model="gpt-3.5-turbo-0301",
+        messages=messages,
+        temperature=0.0
     )
     response_content = response.choices[0].message.content
-    if mode == 'code': response_content = clean_code_string(response_content)
+    if mode == 'code':
+        response_content = clean_code_string(response_content)
+    logging.debug(response_content)
     return response_content
 
+
 def containerize_code(code_string):
     code_string = code_string.replace('your_openai_api_key_here', OPENAI_KEY)
     # uncomment this if you wish to easily use photos from Unsplash API
-    # code_string = code_string.replace('your_unsplash_access_key_here', UNSPLASH_ACCESS_KEY)
+    code_string = code_string.replace(
+        'your_unsplash_access_key_here', UNSPLASH_ACCESS_KEY)
```

Comment:
We want to keep unsplash API features off by default -- they make engshell harder to use and only add value if you are doing tasks with stock photos on the fly